src/nemos/regularizer.py

```diff
@@ -634,7 +489,7 @@ def _get_proximal_operator(
             intercept term is not regularized.
         """
 
-        def prox_op(params, regularizer_strength, scaling=1.0):
+        def prox_op(params, regularizer_strength, scaling=0.5):
```

Comment:
maybe the scaling should be 1.0 because the penalization doesn't have a 0.5 multiplied to the L1 loss. We should check on a paper or book that derives the proximal operator for a group lasso to see what's the analytical form of the operator given the L1 penalization.